src/anemoi/training/train/train.py

```diff
@@ -390,7 +420,8 @@ def train(self) -> None:
 
 @hydra.main(version_base=None, config_path="../config", config_name="config")
 def main(config: DictConfig) -> None:
-    AnemoiTrainer(config).train()
+    trainer = AnemoiTrainer(config)
```

Comment:
Do you need the trainer object?

Reply:
This was a debugging leftover, will remove